libs/libtatsu/Makefile

```
@@ -0,0 +1,71 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libtatsu
+PKG_VERSION:=1.0.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://github.com/libimobiledevice/libtatsu/releases/download/$(PKG_VERSION)
+PKG_HASH:=4f69452d23e50e0ffbe844110e6ab6a900d080e051fbda3b7d595f679dee9bc5
+
+PKG_MAINTAINER:=Rosen Penev <rosenp@gmail.com>
+PKG_LICENSE:=LGPL-2.1-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libtatsu/Default
+  TITLE:=Handle communication with Apple's Tatsu Signing Server (TSS)
+  URL:=https://www.libimobiledevice.org/
+  SUBMENU:=libimobiledevice
+endef
+
+define Package/libtatsu/Default/description
+  libtatsu is part of the libimobiledevice project.
+  The main purpose is to allow creating TSS request payloads,
+  sending them to Apple's TSS server, and retrieving and processing the
+  response. The code was originally part of idevicerestore and has been split
+  out to its own library, while obviously idevicerestore will remain the main
+  consumer.
+endef
+
+define Package/libtatsu
+  $(call Package/libtatsu/Default)
+  SECTION:=libs
+  CATEGORY:=Libraries
+  DEPENDS:=+libcurl +libplist
+  LICENSE:=LGPL-2.1-or-later
+  LICENSE_FILES:=COPYING
+endef
+
+define Package/libtatsu/description
+  $(call Package/libtatsu/Default/description)
+endef
```

```suggestion
define Package/libtatsu
  SECTION:=libs
  CATEGORY:=Libraries
  SUBMENU:=libimobiledevice
  TITLE:=Handle communication with Apple's Tatsu Signing Server (TSS)
  URL:=https://www.libimobiledevice.org/
  DEPENDS:=+libcurl +libplist
endef

define Package/libtatsu/description
  libtatsu is part of the libimobiledevice project.
  The main purpose is to allow creating TSS request payloads,
  sending them to Apple's TSS server, and retrieving and processing the
  response. The code was originally part of idevicerestore and has been split
  out to its own library, while obviously idevicerestore will remain the main
  consumer.
endef
```

Much better and simpler, dont you think? :-)

---

@BKPepe 
Thank you! I will update the PR shortly. Would you like me to also apply the same changes to `libimobiledevice-glue`?

---

Sure, please do that.

---

Ready